src/toil/test/src/resumabilityTest.py

```diff
@@ -34,7 +34,7 @@ def test(self):
         Tests that a toil workflow that fails once can be resumed without a NoSuchJobException.
         """
         options = Job.Runner.getDefaultOptions(self._getTestJobStorePath())
-        options.logLevel = "INFO"
+        options.logLevel = "DEBUG"
```

Comment:
For the next few changes, same as above

Reply:
I'd keep this as info, just to reduce logging.